lstchain/reco/r0_to_dl1.py

```diff
@@ -121,12 +121,52 @@ def setup_writer(writer, subarray, is_simulation):
             writer.exclude(f'telescope/parameters/{tel_name}', 'mc_core_distance')
 
 
+def _camera_distance_to_angle(distance, focal_length):
+    '''Convert a distance in the camera plane into an angle
+
+    This should be replaced by calculating image parameters in
+    telescope frame.
+    '''
+    return np.rad2deg(np.arctan(distance / focal_length))
+
+
+def parametrize_image(image, peak_time, signal_pixels, camera_geometry, focal_length, dl1_container):
+    '''
+    Calculate image parameters and fill them into ``dl1_container``
+    '''
+
+    geom_selected = camera_geometry[signal_pixels]
+    image_selectecd = image[signal_pixels]
+    hillas = hillas_parameters(geom_selected, image_selectecd)
+
+    # Fill container
+    dl1_container.fill_hillas(hillas)
+
+    # convert ctapipe's width and length (in m) to deg:
+
+    for key  in ['width', 'width_uncertainty', 'length', 'length_uncertainty']:
+        value = getattr(dl1_container, key)
+        setattr(dl1_container, key, _camera_distance_to_angle(value, focal_length))
```

Comment:
shouldn't this conversion be done in ctapipe directly?
Having them in deg rather than meters is really interesting when there are several telescopes types.

Reply:
Ha, I saw https://github.com/cta-observatory/cta-lstchain/issues/886 afterwards
Does it mean these lines will be removed once moving to ctapipe 0.12?

Reply:
See comment above in the function doing that ;)